### PR TITLE
PXB-2668 Return the exit code for each pipe when using xbcloud (2.4)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified: source/xbcloud/xbcloud.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst [hunk 1]
```diff
@@ -10,7 +10,15 @@ name. *xbcloud* accepts input via a pipe from xbstream so that it can be
 invoked as a pipeline with *xtrabackup* to stream directly to the cloud without
 needing a local storage.
 
-*xbcloud* stores each chunk as a separate object with a name
+.. note::
+
+   In a Bash shell, the ``$?`` parameter returns the exit code from the last binary. If you use pipes, the ``${PIPESTATUS[x]}`` array parameter returns the exit codes for each binary in the pipe string.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: could we add an example here:

```
If you use pipes like ``xtrabackup [options] | xbcloud [option]``, the ``${PIPESTATUS[x]}`` array parameter ...

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: Added to the example code

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst [hunk 2]
```diff
@@ -10,7 +10,18 @@ name. *xbcloud* accepts input via a pipe from xbstream so that it can be
...
+.. note::
+
+   In a Bash shell, the ``$?`` parameter returns the exit code from the last binary. If you use pipes the ``${PIPESTATUS[x]}`` array parameter returns the exit codes for each binary in the pipe string. 
+
+   .. code-block::
+
+      $ xtrabackup --backup --stream=xbstream --target-dir=/storage/backups/ | xbcloud put [options] full_backup
+       ...
+      $ ${PIPESTATUS[x]}
+      0 0
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]:
```suggestion
> true | false
> echo $?
> 1

# With PIPESTATUS
> true | false
> echo ${PIPESTATUS[0]} ${PIPESTATUS[1]}
> 0 1

```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst (hunk 2 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst]: Added.

**Review by altmannmarcelo — APPROVED** [review]

**Pull request merged**